packages/driver/src/dom/jquery.js

```diff
@@ -31,7 +31,14 @@ const isJquery = (obj) => {
   // does it have the jquery property and does this
   // instance have a constructor with a jquery property
   // on its prototype?
-  return hasJqueryProperty && !!_.get(obj, 'constructor.prototype.jquery')
+  const prop = _.get(obj, 'constructor.prototype.jquery')
+
+  // an HTML element with id="jquery" will show up
+  // as the jquery property of the window constructor
+  // note: instanceof HTMLElement is ineffective
+  // on objects from a different window/frame
+  // so the nodeType and nodeName is used to check if it is an HTML element
+  return hasJqueryProperty && !!prop && prop.nodeType !== 1 && typeof prop.nodeName !== 'string'
```

Comment:
can you explain why `obj && ('jquery' in obj || obj.jquery)` is evaluating to true? That's what was causing the bug right?

Reply:
For any element, the id becomes a property of the window: https://2ality.com/2012/08/ids-are-global.html.

In this case, we will falsely detect that `window.jquery` is the jquery library when it is actually this element.

Reply:
can we just test if `hasJqueryProperty && typeof prop === 'string'` ?

Reply:
Is prop a string when it is actually jquery? I can run some tests tomorrow

Reply:
I believe its the version number string

Reply:
Oh yeah I do remember that, I guess that would work as well if you would rather it be that way

Reply:
makes more sense to me, if we know .jquery must be a string no sense testing if it's not an element